src/panoramic/cli/config/storage.py

```diff
@@ -0,0 +1,29 @@
+import os
+from typing import Any, Dict
+
+from panoramic.cli.file_utils import read_yaml, write_yaml
+from panoramic.cli.paths import Paths
+
+
+def update_config(section: str, data: Dict[str, Any]) -> None:
+    if section != '':
+        data = {section: data}
+
+    config_file = Paths.config_file()
+    if os.path.isfile(config_file):
```

Comment:
```suggestion
    if config_file.is_file():
```
Can we stick to using pathlib where we can?